velox/exec/tests/TableWriteTest.cpp

```diff
@@ -1006,9 +1006,14 @@ class PartitionedTableWriterTest
 
   static std::vector<uint64_t> getTestParams() {
     std::vector<uint64_t> testParams;
-    const std::vector<bool> multiDriverOptions = {false, true};
-    // Add Parquet with https://github.com/facebookincubator/velox/issues/5560
+    std::vector<bool> multiDriverOptions = {false, true};
     std::vector<FileFormat> fileFormats = {FileFormat::DWRF};
+    if (hasWriterFactory(FileFormat::PARQUET)) {
+      fileFormats.push_back(FileFormat::PARQUET);
+      // Add Parquet with https://github.com/facebookincubator/velox/issues/5560
+      // Temporarily remove multiDriver == true from the Parquet run.
+      multiDriverOptions.pop_back();
```

Comment:
Won't this remove the multiDriver option for DWRF as well?
We should simply add Parquet here and skip below inside the loop when 
`multiDriver = true` and `fileFormat = Parquet`.
Can you also update the [issue](https://github.com/facebookincubator/velox/issues/5560) with the error when the above condition holds?

Reply:
Yeah, I tried to be smarter and failed.